symphony-bdk-core/src/main/java/com/symphony/bdk/core/auth/jwt/JwtHelper.java

```diff
@@ -84,6 +103,31 @@ else if (pemPrivateKey.contains(PEM_RSA_PRIVATE_START)) {
 		}
 	}
 
+  /**
+   * Validates a jwt against a certificate.
+   * @param jwt
+   * @param certificate string of the X.509 certificate content in pem format.
+   * @return the content of jwt clain "user" if jwt is successfully validated.
+   * @throws AuthInitializationException if certificate or jwt are invalid.
+   */
+  public static UserClaim validateJwt(String jwt, String certificate) throws AuthInitializationException {
+    System.out.println(certificate);
+    final X509VerificationKeyResolver x509VerificationKeyResolver = new X509VerificationKeyResolver(
+        parseX509Certificate(certificate));
+
+    JwtConsumer jwtConsumer = new JwtConsumerBuilder()
+        .setVerificationKeyResolver(x509VerificationKeyResolver)
+        .setSkipAllValidators()
+        .build();
+
+    try {
+      final Object user = jwtConsumer.processToClaims(jwt).getClaimValue("user");
```

Comment:
It is possible to get claims using `jjwt` that we already import : https://stackoverflow.com/a/54339865

No need to import additional `org.bitbucket.b_c:jose4j`